src/VisualStudio/Core/Def/ID.RoslynCommands.cs

```diff
@@ -33,6 +33,7 @@ public static class RoslynCommands
             public const int RemoveSuppressions = 0x0121;
 
             public const int GoToImplementation = 0x0200;
+            public const int GoToBase = 0x0201;
```

Comment:
As mentioned earlier, any changes in this file shouldn't be necessary if the command is implemented on the editor side.